### DISCUSSION TITLE
Welcome to MSIX-PackageSupportFramework Discussions!

### DISCUSSION THREAD

**Discussion by TimMangan** [TimMangan/MSIX-PackageSupportFramework#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

This discussion area is separate from the main fork of the PSF project that Microsoft maintains, which is different than the Issues area that only exists on the Microsoft fork.  So please use this discussion area rather than issues!

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mathewhunter** [comment]
Just tried it today, I've been using the old PSF for various things, mostly running scripts and providing arguments... Their file fixup never really seemed to do what I needed.  Your MFRFixup is amazing though.  After struggling with PSF for hours on what should have been a simple fix (child process falling outside the bubble and then couldn't find files in the bubble), MFRFixup fixed it with basically no config.  Amazing!  Thanks for all your hard work Tim this community is lucky to have you.

**Reply by TimMangan** [reply]
Thanks!
